users/serializers.py

```diff
@@ -322,6 +327,83 @@ class Meta:
         )
 
 
+class ChangeEmailRequestCreateSerializer(serializers.ModelSerializer):
```

Comment:
Could you add some tests for these things:

- The custom validations in `validate()`
- The task call that occurs in `create()` (mock the task)

Reply:
These tests already covered on Friday's commits.